lib/registry.js

```diff
@@ -16,7 +17,9 @@ export function define(id, spec) {
   if (get(id)) {
     console.warn(`Overwriting spec ${getName(id)}`)
   }
-  spec.name = getName(id)
+  if (!spec.name && typeof spec !== "symbol") {
```

Comment:
Another thing :) :
```  if (get(id)) {
    console.warn(`Overwriting spec ${getName(id)}`)
  }
```
Is this right for aliases? Let's say "alias" points to "id" which points to spec. Later I want to define "alias" to point to "id2", wich points to spec2. It's incorrect to say it will overwrite spec, as "id" still validly points to it.

Perhaps the define() should directly check REGISTRY for an overwrite instead of using get()?